README.md

```diff
@@ -120,6 +121,17 @@ func main() {
 
 	// Print a day
 	fmt.Println(randomdata.Day())
+  
+	// Print a month
+	fmt.Println(randomdata.Month())
+  
+	// Print date like Thursday 22 Aug 2016
+	timestamp := time.Now()
```

Comment:
Perhaps it would make sense to also wrap this into a convenient method, seems to me as this would be a fairly standard use case.